docker/test/docker-compose.yml

```diff
@@ -68,10 +65,9 @@ services:
       SPROCKETS_CACHE: /cache
       BLOG: https://mampf.blog
     volumes:
-        - type: bind
-          source: ../../spec/
-          target: /usr/src/app/spec/
-        - ../../coverage:/usr/src/app/coverage
+      - ../../:/usr/src/app/
+      - ../../coverage:/usr/src/app/coverage/
+      - "/usr/src/app/public/"
```

Comment:
Please don't ask, this took the whole night to figure out as I always had to wait for 10 minutes before I could check if any changes worked 😭

Reply:
Thanks for the effort! Works like a charm.